.github/workflows/python-bindings-publish.yml

```
@@ -0,0 +1,59 @@
+name: Build and Upload Wheels for the Python Bindings
+
+on:
+  push:
+    branches:
+      - dev
+    paths:
+      - '.github/workflows/python-bindings-publish.yml'
+      - '.github/actions/**'
+      - 'bindings/python/**'
+      # Watch the Rust core too
+      - 'src/**'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+
+jobs:
+  wheels:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

```suggestion
        os: [ubuntu-18.04, macos-latest, windows-latest]
```

I think we build on Ubuntu 18.04 to avoid issues with glibc (#588)

---

Fair enough.